### PR TITLE
#545 Support for IntelliJ IDEA 2020.1

### PR DESCRIPTION
**Pull request by hakuseki**
I'm a little unsure on how to complete this but I followed the guidelines and updated some resources and fixed the AllIcons compilation error.

If I have missed something or done wrong, please let me know and Iäll fix it

### CODE REVIEW

**Comment by hakuseki** [comment]
Damn, I see my editor made some structural changes to the code.
If this is a problem then I can revert everything and do over?

File: gradle.properties [hunk 1]
```diff
@@ -1 +1,2 @@
-camelVersion = 2.24.2
\ No newline at end of file
+camelVersion  = 2.25.1
+intellij_type = IU
```
Comment (fharms) [inline gradle.properties:2]: Please remove this. It is default to IC, but we have make it possible to override locally

File: camel-idea-plugin/src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <idea-plugin url="https://github.com/camel-idea-plugin/camel-idea-plugin">
   <id>org.apache.camel</id>
   <name>Apache Camel IDEA Plugin</name>
-  <version>0.6.2</version>
+  <version>0.6.5</version>
```
Comment (fharms) [inline camel-idea-plugin/src/main/resources/META-INF/plugin.xml:4]: Need to be changed to v0.6.4

File: camel-idea-plugin/src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -30,6 +30,10 @@
     ]]></description>
 
   <change-notes><![CDATA[
+      v0.6.5
+      <ul>
+      <li>Upgrade to IntelliJ IDEA 2020.1</li>
```
Comment (fharms) [inline camel-idea-plugin/src/main/resources/META-INF/plugin.xml:35]: Should be moved to the v0.6.4. The next version we release is v0.6.4

File: camel-idea-plugin/src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (hakuseki) [reply camel-idea-plugin/src/main/resources/META-INF/plugin.xml:4]: I figured that but then I saw some change logs on 0.6.4. I'll set it to 0.6.4

**Comment by fharms** [comment]
@hakuseki I will merge the PR later today and release it later this weekend

**Pull request merged**